frontend/viewer/src/lib/entry-editor/object-editors/EntryEditorPrimitive.svelte

```diff
@@ -35,7 +35,7 @@
   }
 </script>
 
-<Editor.SubGrid class="gap-2" style="grid-template-areas: {objectTemplateAreas($currentView, entry)}">
+<Editor.SubGrid class="gap-2 editor-primitive" style="grid-template-areas: {objectTemplateAreas($currentView, entry)}">
```

Comment:
I don't know how but we should document why these are here as it's very non obvious, I could easily see us removing them because there isn't any associated styles. Maybe we just use a const string value and reference it from here and in EntryEditor?

Reply:
I defined a const in `EntryEditor` and pass it in, so everything's explicit in the same file.